python/ray/_private/worker.py

```diff
@@ -2620,10 +2643,57 @@ def get(
         return values
 
 
+@PublicAPI
+def _write_channel(value: Any, object_ref: ObjectRef, num_readers: int):
+    worker = global_worker
+    worker.check_connected()
```

Comment:
Btw this call is pretty expensive. Maybe we should skip it. 

In my macbook it takes 10us